src/utils.ts

```diff
@@ -72,7 +72,63 @@ export function makeIncRules(
   }
 }
 
+/** Turns a high-level `{ sm: 0, md: 200 }` breakpoint config into a useful set of multiple media queries. */
+export function makeBreakpoints(
+  breakpoints: Record<string, number>
+): Record<string, string> {
+  const r: Record<string, string> = {};
+  const bps = Object.keys(breakpoints);
+  Object.entries(breakpoints).forEach(([bp, px], i) => {
+    const isFirst = i === 0;
+    const isLast = i === bps.length - 1;
+    // Calc this breakpoint's min/max, which is its px --> the next bp's px - 1
+    const min = !isFirst ? `${px}px` : "0";
+    const max = !isLast ? `${breakpoints[bps[i + 1]] - 1}px` : "0";
```

Comment:
Do you need these ternary statements? Based on the logic below it doesn't seem like `min` is ever used if `isFirst`, and `max` is never used if `isLast`,... I'm pretty sure?